MSAL/xcconfig/msal__static__lib__ios.xcconfig

```diff
@@ -3,9 +3,7 @@
 // Force the linker to resolve symbols.
 GENERATE_MASTER_OBJECT_FILE = YES
 
-// Add armv7s and arm64e to standard ARCHs.
-ARCHS = $(ARCHS_STANDARD) armv7s arm64e
```

Comment:
why are these removed ?

Reply:
armv7s is a 32-bit architecture no longer supported. arm64e is unused. $(ARCHS_STANDARD) is the default